src/terrain.hpp

```diff
@@ -11,6 +11,8 @@
 #include <stdint.h>
 #include <map>
 
+#include <glm/glm.hpp>
```

Comment:
Again, you're not using opengl

Reply:
I added glm then removed it, but forgot to remover the include.